test/e2e/app-dir/dynamic/app/dynamic/named-export/page.js

```diff
@@ -2,7 +2,7 @@ import dynamic from 'next/dynamic'
 
 const Button = dynamic(() =>
   import('./client').then((mod) => {
-    return { default: mod.Button }
+    return mod.Button
```

Comment:
we should be testing both usages here, i.e.
- `.then((mod) => mod.Button)`
- `.then((mod) => ({ default: mod.Button })`